.test.sh

```diff
@@ -2,4 +2,5 @@
 set -euo pipefail
 
 python -m mypy .
```

Comment:
shall we remove this then?

Reply:
@shahinism Mypy do better job with type checking. I feel like keeping it half enable is still better than not having it at all.